tests/parser_templates/cli/show_version/main.yaml

```diff
@@ -1,4 +1,22 @@
 ---
+### IOS 15.1(4)M4
```

Comment:
do you mind moving this back to original location, this helps keep the diff smaller and easier to review.

Reply:
The tests are in ascending order now, which I think is more important than diff size. Hopefully there will be more test samples from others in the future.